srcs/vm/op_arch_checker.c

```diff
@@ -41,5 +41,12 @@ bool is_valid_enbyte(t_inst inst, t_enbyte enbyte)
 		ret = is_valid_arg(inst, enbyte.arg2, 2);
 	if (ret)
 		ret = is_valid_arg(inst, enbyte.arg3, 3);
+	if (ret)
```

Comment:
Dit is al gefixed volgens mij in de nieuwe master, scuzii had het er net ff met floris over die dacht dat hij dat moest doen en vroeg of ik het ff kon doen, dubbel check even op master